app/routes/wallet/dids.py

```diff
@@ -19,12 +19,30 @@
 router = APIRouter(prefix="/v1/wallet/dids", tags=["wallet"])
 
 
-@router.post("", response_model=DID)
+@router.post("", response_model=DID, summary="Create Local DID")
 async def create_did(
     did_create: Optional[DIDCreate] = None,
     auth: AcaPyAuth = Depends(acapy_auth_from_header),
 ):
-    """Create Local DID."""
+    """
+    Create Local DID
+    ---
+
+    This endpoint allows you to create a new DID in the wallet.
+    The `method` parameter is optional and can be set to 'key' or 'sov'.
+
+    Request body:
+    ---
+        DIDCreate:
+            method: Optional[str]
+            options: Optional[DIDCreateOptions]
```

Comment:
There's one "gotcha" to keep in mind with the `DIDCreateOptions`

They are optional, but it contains a required field. So, if a client passes `.., options: {}, ..`, then they'll get a validation error, since `key_type` is required and must be one of `["ed25519", "bls12381g2"]`.

Maybe we can streamline this somehow ... instead of us accepting `options: Optional[DIDCreateOptions]`, let's just expose the 2 fields that it contains separately. So we expose:
```
    did: Optional[Annotated[str, Field(strict=True)]] = Field(
        default=None,
        description="Specify final value of the did (including did:<method>: prefix)if the method supports or requires so.",
    )
    key_type: StrictStr = Field(
        description="Key type to use for the DID keypair. Validated with the chosen DID method's supported key types."
    )
```

And we have a default for key_type = ed25519, indicating it's required, and did is optional if they want to set its final value.

I think unwrapping the options field like that will make things a bit clearer for the client

Then of course we have to think about backward compatibility, since replacing options is a breaking change.
So, see if you can keep options - mark it as deprecated - but add `did` and `key_type` as additional args, which configures the DIDCreateOptions we pass to acapy, unless the deprecated `options` field was passed, in which case we use that. Later we sunset and remove the options field.